### ISSUE TITLE
(not an issue, a question) Can I get spectral transmission/reflective data for buildup - as if using Optics?

### ISSUE THREAD

**Issue by aegis1980** [LBNL-ETA/pyWinCalc#35]
Is there a function to access the spectral transmission & reflection data for a buildup - as one might generate in Optics? e.g. wavelength vs transmission for an igu.

I have had a good* look around the c++ code and couldn't find it.   

* my best effort!

**Comment by vidanovic** [comment]
@aegis1980 You are correct. I did not make an interface to retrieve spectral data for the IGU. Will put onto the list and hopefully implement soon enough.

**Comment by aegis1980** [comment]
Thanks for getting back to me. 
Might try my own implementation - a 'lo-fi' version - and see how close I get. 
Keep up good work (from a Facade Engineer)! Appreciate it.
